File: lib/java/src/com/eliudarudo/cliapp/player/Song.java
package com.eliudarudo.cliapp.player;

public class Song {
    String title;
    String artist;
    int rating;

    public Song(String title, String artist, int rating) {
        this.title = title;
        this.artist = artist;
        this.rating = rating;
    }

    public void print() {
        String title = Utility.printWithBuffer(this.title, 20);
        String artist = Utility.printWithBuffer(this.artist, 27);
        System.out.println(title + " " + artist + " " + this.rating);
    }
}

File: server/src/lib/nodejs/src/player/song.js
const { printWithBuffer } = require("./utility")

module.exports.Song = class {
    constructor(title, artist, rating) {
        this.title = title
        this.artist = artist
        this.rating = rating
    }

    print() {
        const title = printWithBuffer(this.title, 20)
        const artist = printWithBuffer(this.artist, 27)
        console.log(`${title} ${artist} ${this.rating}`)
    }
}